raster_analysis/tiling.py

```diff
@@ -181,3 +198,28 @@ def _get_rounded_bounding_box(
             geom.bounds[2] + (-geom.bounds[2] % width),
             geom.bounds[3] + (-geom.bounds[3] % width),
         )
+
+
+    def _get_cache_key(self, tile: Polygon) -> str:
```

Comment:
could you move this function to the result_stores module? I was trying to keep any of the dynamodb stuff contained to there

Reply:
That makes sense.. moving it.